internal/driver/types.go

```diff
@@ -73,3 +80,25 @@ var PixelFormatV4l2Mappings = map[string]uint32{
 	"Y8I":  PixFmtY8I,
 	"Y12I": PixFmtY12I,
 }
+
+var StreamFormatTypeMap = map[uint32]string{
+	v4l2.PixelFmtRGB24: RGB,
+	v4l2.PixelFmtGrey:  Greyscale,
+	v4l2.PixelFmtYUYV:  RGB,
+	v4l2.PixelFmtMJPEG: RGB,
+	v4l2.PixelFmtJPEG:  RGB,
+	v4l2.PixelFmtMPEG:  RGB,
+	v4l2.PixelFmtH264:  RGB,
+	v4l2.PixelFmtMPEG4: RGB,
+	v4l2.PixelFmtUYVY:  Greyscale,
+	PixFmtBYR2:         RGB,
+	PixFmtY8I:          Greyscale,
+	PixFmtY12I:         Greyscale,
+	PixFmtDepthZ16:     Depth,
+}
+
+const (
```

Comment:
I would put these back where they were, or move the `type` definition down here. I like them being together.